docs/release-notes/NuGet-6.13.md

```diff
@@ -23,6 +16,104 @@ NuGet distribution vehicles:
 
 <sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
 
-## Not yet released
+## Summary: What's New in 6.13
+
+* Detect if restore used NuGetAudit or not for PackageReference projects - [#13778](https://github.com/NuGet/Home/issues/13778)
```

Comment:
We probably don't need this as a high level item and we can just move it to the issues fixed.